### PR TITLE
Fix the contact info margin

### LINKED ISSUE #30539: Gutenpack: Contact Info front-end margin
The block needs some sort of generic margin-bottom (e.g. `1.5em`) to make sure it has some spacing by default without to rely on the theme's CSS.

screenshot 2019-02-01 at 10 10 07

In a way, it's similar to #30436

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request

* This adds a margin to the contact info block as suggested by @thomasguillot here: #30539

#### Testing instructions

* Check that the contact info block has a bottom margin
* Not sure how to best test this

Fixes #30539

### CODE REVIEW

**Comment by enejb** [comment]
Worked as expected for me. Nicely done.

File: client/gutenberg/extensions/contact-info/view.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/** @format */
```
Comment (sirreal) [inline client/gutenberg/extensions/contact-info/view.js]: NIT: We don't need these any more. Prettier always runs on JS files now.
```suggestion
```

File: client/gutenberg/extensions/contact-info/view.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (sirreal) [inline client/gutenberg/extensions/contact-info/view.js]:
```suggestion
```

File: client/gutenberg/extensions/contact-info/style.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.wp-block-jetpack-contact-info {
+	margin-bottom: 1.75em;
```
Comment (thomasguillot) [inline client/gutenberg/extensions/contact-info/style.scss]: Can we change this to `1.5em` so we have some consistency with our other blocks.

Comment (scruffian) [reply client/gutenberg/extensions/contact-info/style.scss]: Done :)

**Pull request merged**